src/poke_env/data/static/moves/gen1moves.json

```diff
@@ -211,7 +211,7 @@
         "priority": 0,
         "secondary": null,
         "target": "self",
-        "type": "???",
+        "type": "Normal",
```

Comment:
Can you remove the changes to this files? These are auto-generated - the issues they cause in gen 1 should be fixed in python rather than the raw data.